podman/Containerfile

```diff
@@ -15,6 +15,19 @@ ARG PACKER_BUILD_NAME=
 ENV AI_PATH=/usr/src/automation_images \
     CONTAINER=1
 
+ARG IMG_SFX=
+ARG CIRRUS_TASK_ID=
+ARG GIT_HEAD=
+# Ref: https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL org.opencontainers.image.url="https://cirrus-ci.com/task/${CIRRUS_TASK_ID}"
+LABEL org.opencontainers.image.documentation="https://github.com/containers/automation_images/blob/${GIT_HEAD}/README.md#container-images-overview-step-2"
+# Renovate is sensitive to the following, they must point at the repo. and reflect commited
+# git sha's and current or future repo. tags/releases.
+# Ref: https://docs.renovatebot.com/modules/datasource/docker/
+LABEL org.opencontainers.image.source="https://github.com/containers/automation_images"
```

Comment:
There are eleven Containerfiles in this repo. Today. Could you add `/podman/Containerfile`?

Reply:
I originally had it this way, but was afraid Renovate will get mad.  Renovate isn't currently used for this image, so I can put it back.  Maybe it'll be okay?